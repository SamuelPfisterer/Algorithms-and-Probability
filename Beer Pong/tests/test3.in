30
6 6 6 3
0.824001 0.517434
4 20 4 3
0.696552 0.978555
7 11 7 3
0.447983 0.795764
6 11 16 3
0.436804 0.720079
4 16 9 3
0.6123 0.622101
3 18 10 3
0.989115 0.660105
5 10 6 3
0.575033 0.749918
2 7 16 3
0.78263 0.797416
4 4 4 3
0.410852 0.967345
4 16 20 3
0.560336 0.611723
3 18 6 3
0.524088 0.969193
5 15 11 3
0.542723 0.704321
3 9 18 3
0.932748 0.530302
3 10 5 3
0.430527 0.5811
6 7 10 3
0.858604 0.529686
7 17 20 3
0.329054 0.872028
2 15 6 3
0.95196 0.785979
6 19 18 3
0.403869 0.60129
2 20 4 3
0.814463 0.756009
4 6 6 3
0.536038 0.870562
5 8 11 3
0.982598 0.635386
5 14 5 3
0.497353 0.559185
2 13 10 3
0.786416 0.510214
3 11 18 3
0.844704 0.926993
3 7 19 3
0.420612 0.898958
7 18 19 3
0.534282 0.602519
5 10 14 3
0.700735 0.828078
7 9 17 3
0.793487 0.897789
6 18 18 3
0.946381 0.526481
6 18 9 3
0.518686 0.984833
