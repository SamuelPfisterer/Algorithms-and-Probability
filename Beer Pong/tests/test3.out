0.0000178
0.0023905
0.0000006
0.0001597
0.0062401
0.1502843
0.0006466
0.7708927
0.0165903
0.0197647
0.0401269
0.0009305
0.3542524
0.0260856
0.0001439
0.0000164
0.4376855
0.0000067
0.2336103
0.0304716
0.0161159
0.00001
0.4960167
0.4894723
0.2007919
0.0000008
0.0256285
0.0117782
0.0001184
0.0002741
